src/main/java/seedu/address/logic/commands/AssignPolicyCommand.java

```diff
@@ -25,58 +26,57 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Assigns the policy to a person.\n"
             + "Parameters: "
-            + PREFIX_POLICY_INDEX + "POLICY INDEX "
-            + PREFIX_PERSON_INDEX + "PERSON INDEX\n"
+            + "INDEX (must be a positive integer) "
+            + PREFIX_POLICY + "POLICY NAME\n"
             + "Example: "
-            + COMMAND_WORD + " "
-            + PREFIX_POLICY_INDEX + "1 "
-            + PREFIX_PERSON_INDEX + "1";
+            + COMMAND_WORD + " 1 "
+            + PREFIX_POLICY + "health insurance";
 
     public static final String MESSAGE_ASSIGN_POLICY_SUCCESS = "Assigned Policy: %1$s to Person: %2$s";
     public static final String MESSAGE_ALREADY_ASSIGNED = "Person: %1$s already has the Policy: %2$s.";
+    public static final String MESSAGE_POLICY_NOT_FOUND = "Policy: %1$s not found in address book.";
```

Comment:
I think we should remove the references to 'address book' so maybe it could be "Policy: %1$s not found in policy list." or something like that.